lib/transition-checker.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+var List = require('immutable').List;
+var Octokat = require('octokat');
+
+/**
+ * @exports lib/transition-checker
+ */
+
+var TransitionChecker = {};
+
+/**
+ * @returns {Promise.<List.<String>>}
+ */
+
+TransitionChecker.check = function (profile, latestVersion, previousVersion, isEditorial, isUpdate) {
+  return new Promise(function (resolve, reject) {
+    var errors = new List();
+
+    if (profile === 'WD' || profile === 'WG-NOTE' || isUpdate) {
+      resolve({ errors: errors, requiresCfE: false });
+    }
+    else if (profile === 'CR') {
+      if (previousVersion.includes('/WD-') || (previousVersion.includes('/CR-') && !isEditorial)) {
+        var octo = new Octokat({
+          token: global.GH_TOKEN
+        });
+        var repo = octo.repos('w3c', 'transitions');
+
+        var shortname = latestVersion.match(new RegExp(/.*\/([^/]+)\/$/))[1];
+        var directorApproval = 'Transition approved.';
+        var commApproval = 'Draft transition approved.';
+        var directorApprovalFound = false;
+        var commApprovalFound = false;
+
+        repo.issues.fetch(
+          {
+            labels: 'Awaiting publication',
+            state: 'open',
+            per_page: 100 // TODO: get all the issues, not just the first 100
```

Comment:
Until we retrieve them all: perhaps add a param [`sort: 'updated'`](https://developer.github.com/v3/issues/#parameters) to make sure we always get the 100 that were most recently updated?

Reply:
Yes, good idea!